mldsa/sign.c

```diff
@@ -14,6 +14,76 @@
 #include "sign.h"
 #include "symmetric.h"
 
+static int mld_check_pct(uint8_t const pk[CRYPTO_PUBLICKEYBYTES],
+                         uint8_t const sk[CRYPTO_SECRETKEYBYTES]);
```

Comment:
This is missing the CBMC contract. 
It should then be proven for the actual implementation - see [this](https://github.com/pq-code-package/mlkem-native/tree/main/proofs/cbmc/check_pct).

Reply:
This will rely on the `crypto_sign_signature`function which is not proven yet, but you can just add a contract for that function and then it should hopefully just go through.

Reply:
I think crypto_sign_signature() is proven... I remember doing it!